README.md

```diff
@@ -1,22 +1,30 @@
 # Viewing Party
```

Comment:
I like that we are working on the ReadMe from the beginning

Reply:
Me too!